extensions/ticktick/src/create.tsx

```diff
@@ -89,14 +102,14 @@ export default function TickTickCreate() {
       }
       isLoading={isLoading}
     >
-      <Form.TextField ref={titleRef} id="title" autoFocus title="Title" placeholder="No Title" />
-      <Form.Dropdown ref={listPickerRef} id="list" title="List" onChange={onListChange} defaultValue={defaultAddList}>
+      <Form.TextField ref={titleRef} id="title" onBlur={onTitleBlur} autoFocus title="Title" placeholder="No Title" />
+      <Form.TextArea ref={descRef} id="desc" title="Description" placeholder="" />
+      <Form.DatePicker ref={dueDatePickerRef} id="dueDate" title="Due Date" type={Form.DatePicker.Type.DateTime} />
```

Comment:
restore the original order

Reply:
My main motivation for that was that if it's being done by autocomplete, then it probably shouldn't be in the next tab-index. What do you think?

(I've put the original order back in place, so we can keep it as it was or change it we decided to)

Reply:
I get your point. However, my concern is that the majority of users may find it inconvenient to have to go to the bottom of the form to change the project.

Reply:
I agree that it is wiser decision to keep it as it is 👍 I assume this thread is also resolved